base/ircd.yaml

```diff
@@ -193,6 +193,7 @@ server:
         # IPs/networks which are exempted from connection limits
         exempted:
```

Comment:
Can we add `*.hashbang.sh` to this? Currently, `users | wc` on our de1 server has 565 entries.

Reply:
Currently this can only be controlled via addresses or CIDRs; using forward-confirmed RDNS isn't supported at this time.